linkerd/app/inbound/src/policy/config.rs

```diff
@@ -1,6 +1,7 @@
 use super::{discover::Discover, DefaultPolicy, ServerPolicy, Store};
+use ahash::AHashMap as HashMap;
 use linkerd_app_core::{control, dns, identity, metrics, svc::NewService};
-use std::collections::{HashMap, HashSet};
+use std::collections::HashSet;
```

Comment:
Would it make sense to swap out `collections::HashSet` for ahash's HashSet wrapper? 

I'm not sure how frequently hashing would be done here since it's unlikely for ports to change that much in this context.

Reply:
this is really just used for configuration and shouldn't factor into application performance. i had tried to change but it forces some annoying changes into `linkerd_app::config`, so i left that out for now.